images/source/Dockerfile

```diff
@@ -8,4 +8,4 @@
 FROM centos:7
 
 ADD origin-local-release.repo /etc/yum.repos.d/
-ADD centos-paas-sig-openshift-future.repo /etc/yum.repos.d/
+ADD centos-paas-sig-openshift-origin36.repo /etc/yum.repos.d/
```

Comment:
So adding this to the source images makes the base image confused about which yum repo to choose. I think we need to handle this on a case-by-case basis. Also I am not sure we need source.